index.js

```diff
@@ -3,30 +3,58 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let i = {};
+  while (i< rolls.length){
+    if(rolls[i] === toequal.rolls)
+    return false
+    i++
+  }
+  return true
+}
+
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  let result = 0
+  for(let value of rolls){
+    result += value;
+  }
+  return null;
+}
 
 /**
  * Returns a new array from the `rolls` array with only values equal to or greater than the `lowest` value.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} lowest - A number that represents the lowest allowed value in the new array.
  * @returns {number[]} An array of all numbers that are equal to or higher than the `lowest` value.
  */
-function filterOutLowValues(rolls, lowest) {}
+function filterOutLowValues(rolls, lowest) {
+  let newArr = []
+  for (let i =1; i < rolls.length; i++){
+    if(rolls[i]< lowest){
```

Comment:
Your conditional is very close. Since you are looking to for "equal to or higher than" you comparison operator should be `=>`.